src/ai/ai_core.cpp

```diff
@@ -293,13 +294,17 @@
 
 /* static */ int AI::GetStartNextTime()
 {
-	/* Find the first company which doesn't exist yet */
-	for (CompanyID c = COMPANY_FIRST; c < MAX_COMPANIES; c++) {
-		if (!Company::IsValidID(c)) return AIConfig::GetConfig(c, AIConfig::SSS_FORCE_GAME)->GetSetting("start_date");
+	int start_next = _settings_game.ai.ai_start_next;
+	if (start_next > 0 && AI::START_NEXT_DEVIATION != 0) {
+		/* Add random deviation */
+		start_next += InteractiveRandomRange(AI::START_NEXT_DEVIATION * 2 + 1) - AI::START_NEXT_DEVIATION;
+
+		/* ai_start_delay = 0 is a special case, where random deviation does not occur.
+		 * If ai_start_delay was not already 0, then a minimum value of 1 must apply. */
+		start_next = Clamp(start_next, std::max((int)AI::START_NEXT_MIN, 1), AI::START_NEXT_MAX);
```

Comment:
`AI::START_NEXT_MIN` is 0, so std::max will always return 1

Reply:
Same as above.